server/dba/migrations/20-forests.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+module.exports = {
+  up: function(queryInterface, Sequelize) {
+    return queryInterface.createTable('forests', {
+      id: { type: Sequelize.INTEGER, primaryKey: true, field: 'id' },
+      forestName: { type: Sequelize.STRING(50), field: 'forest_name' },
+      description: { type: Sequelize.STRING(1000), field: 'description' },
+      forestUrl: { type: Sequelize.STRING, field: 'forest_url' },
+      treeHeight: { type: Sequelize.INTEGER, field: 'tree_height' },
+      stumpHeight: { type: Sequelize.INTEGER, field: 'stump_height' },
+      stumpDiameter: { type: Sequelize.INTEGER, field: 'stump_diameter' },
+      startDate: { type: Sequelize.DATE, field: 'start_date' },
+      endDate: { type: Sequelize.DATE, field: 'end_date' },
+      createdAt: { type: Sequelize.DATE, defaultValue: Sequelize.NOW, allowNull: false, field: 'created' },
+      updatedAt: { type: Sequelize.DATE, defaultValue: Sequelize.NOW, allowNull: false, field: 'updated' }
+    });
+  },
+  down: function(queryInterface, Sequelize) {
+    return queryInterface.dropTable('forests');
```

Comment:
Why are there dropTables being used instead of altering the columns. Please take a look at the earlier migrations and the `doTransaction` method that promises multiple changes.

Reply:
@lpendem ok, please drop the story link when complete.

Reply:
@lauraGgit here it is: https://github.com/nciinc/fs-intake-module/issues/78